app/controllers/user.controller.js

```diff
@@ -13,7 +13,7 @@ signup = async(req, res) => {
   const { user } = req;
   const { email } = user;
 
```

Comment:
According to the doc challenge, you should add the username field and pass it when you want to create a user. Kindly read and follow the doc challenge requirements.